CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ Future Todo List
 - Update to use latest v2 PIE design tokens
 
 
+v10.0.0
+------------------------------
+*September 1, 2022*
+
+### Changed
+- **Breaking** Upgraded `@justeat/pie-design-tokens` to `v2.0.0`
```

Comment:
Worth linking to the pie-design-tokens changelog here, as that'll tell people what's changed that they need to be aware of.